tests/pipelines/marigold/test_marigold_normals.py

```diff
@@ -139,6 +139,38 @@ def get_dummy_inputs(self, device, seed=0):
         }
         return inputs
 
+    def test_attention_slicing_forward_pass(self):
+        self._test_attention_slicing_forward_pass(test_mean_pixel_difference=False)
+
+    def test_marigold_normals(self):
+        device = "cpu"
+
+        components = self.get_dummy_components()
+        pipe = self.pipeline_class(**components)
+        pipe.to(device)
+        pipe.set_progress_bar_config(disable=None)
+
+        inputs = self.get_dummy_inputs(device)
+        image = pipe(**inputs).prediction
+        image_slice = image[0, -1, -3:, -3:]
+
+        self.assertEqual(image.shape, (1, 3, 32, 32))
```

Comment:
@toshas 
this is an issue that for marigold pipeline numpy (`np`) output is channel first 
np output should always be channel last and torch tensor output should always be channel first - can we update the pipeline so that it output the expected output and also update image processor so that it's able to handle the expected np input?

Reply:
Yep, I cherry-picked your commits, this issue is fixed now - thanks!